second-semester/Filesystem/src/main/java/ua/knu/cli/command/Command.java

```diff
@@ -1,13 +1,14 @@
 package ua.knu.cli.command;
 
 import lombok.val;
+import ua.knu.exceptions.FileOperationException;
 import ua.knu.util.Constants;
 
 public abstract class Command {
 
     public abstract boolean canProcess(String command);
 
-    public abstract void process(String command);
+    public abstract void process(String command) throws FileOperationException;
```

Comment:
Why abstract command class throws specific for file interaction exception, this breaks command abstraction

Reply:
Use SneakyThrows instead

Reply:
If we don't add this throw, we'll have: `Unhandled exception: ua.knu.exceptions.FileOperationException`